src/parse.ts

```diff
@@ -2,7 +2,7 @@ import remarkFrontmatter from 'remark-frontmatter';
 import remarkGFM from 'remark-gfm';
 import remarkParse, { RemarkParseOptions } from 'remark-parse';
 // @ts-expect-error
-import remarkSlug from 'remark-slug';
+// import remarkSlug from 'remark-slug';
```

Comment:
I'll need to port it over here, I believe.
It's CJS and Rollup doesn't handle it too well.